### PR TITLE
refactor: change spacing on page-content-layout

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

The `PublicPageLayout` is used internally mainly on the login pages. With the redesign the text alignment on the header and the spacing on the footer (legal message) should change.

There are different ways to achieve this:

1. Add `props` such as `welcomeMessageTextAlignment` and `legalMessageScale`
2. Using `themedValue` for old and new spacings

I wanted to suggest 2. first. I don't see any reason we should allow different text alignments in the header nor different paddings of the legal message to the top.

CleanShot 2023-03-08 at 14 31 51@2x

CleanShot 2023-03-08 at 15 07 45@2x

### CODE REVIEW

**Review by emmenko — APPROVED** [review]
👌

File: packages/application-components/src/components/public-page-layout/public-page-layout.tsx [hunk 1]
```diff
@@ -78,16 +80,16 @@ const PublicPageLayout: FC<TProps> = (props) => {
...
+                  text-align: ${themedValue('left', 'center')};
                 `}
               >
                 {props.welcomeMessage}
               </div>
             </Text.Headline>
           </ContainerColumn>
         )}
-        <Spacings.Stack scale="s">
+        <Spacings.Stack scale={themedValue('xs', 'l')}>
```
Comment (CarlosCortizasCT) [inline packages/application-components/src/components/public-page-layout/public-page-layout.tsx:91]: I wanted to double check if the changed value for the default theme is also intentional.

Comment (tdeekens) [reply packages/application-components/src/components/public-page-layout/public-page-layout.tsx:91]: Nope. An 🦅 👁️ as usual. Thanks!

**Review by CarlosCortizasCT — APPROVED** [review]
👍

**Pull request merged**